### ISSUE TITLE
Label accessibility with ARIA W3C Recommendations

### ISSUE THREAD

**Issue by nikiluk** [OpenEthicsAI/OETP#13]
Currently, the embedded label icons are not accessible for speech narration and they don't support ARIA accessibility attributes.
